frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/local-volume-set-inner.tsx

```diff
@@ -234,6 +249,9 @@ type LocalVolumeSetInnerProps = {
   dispatch: React.Dispatch<Action>;
   diskModeOptions?: { [key: string]: string };
   allNodesHelpTxt?: string;
+  taintsFilter?: (nodes: NodeKind) => boolean;
```

Comment:
```suggestion
  OCStaintFilter?: (nodes: NodeKind) => boolean;
```

Reply:
I feel we should keep it generic and not use any OCS based terms in lso.

Reply:
ack